Octokit.Reactive/Clients/IObservableRepositoryContentsClient.cs

```diff
@@ -6,23 +6,40 @@ namespace Octokit.Reactive
     /// <summary>
     /// Client for accessing contents of files within a repository as base64 encoded content.
     /// </summary>
+    /// <remarks>
+    /// See the <a href="https://developer.github.com/v3/repos/contents/">Repository Contents API documentation</a> for more information.
+    /// </remarks>
     public interface IObservableRepositoryContentsClient
     {
         /// <summary>
         /// Returns the HTML rendered README.
         /// </summary>
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
-        /// <returns></returns>
+        /// <returns>A <see cref="IObservable{Readme}"/> of <see cref="Readme"/> representing the README.md at the specified repository.</returns>
```

Comment:
Let's just remove the whole `<return></return>` elements wherever we're touching methods in these files.